librosa/core/constantq.py

```diff
@@ -369,9 +351,12 @@ def hybrid_cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
                                          filter_scale=filter_scale)
 
     # Determine which filters to use with Pseudo CQT
-    pseudo_filters = lengths < 2*hop_length
+    # These are the ones that fit within 2 hop lengths after padding
+    pseudo_filters = 2.0**np.ceil(np.log2(lengths)) < 2 * hop_length
```

Comment:
The old test was missing filters in some edge cases, leading to strong discontinuities.

I think what matters is how __fft_response is going to behave when provided that set of filters as input; in that sense, the two tests should match up.